### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by acrolink** [mkloubert/vs-script-commands#5]
Not sure if this has something to do with latest VC Code releases, but the commands don't get registered. A short-cut I defined previously to call a custom command results now in:

`command my_command not found`

**Comment by mkloubert** [comment]
Can you post a sample configuration?

**Comment by acrolink** [comment]
```
    "script.commands": {
        "commands": {
            "id": "acro.jameel",
            "script": "C:/Users/User/.vscode/my_scripts/jameel.js"
        }
    },
```
The `js` file:

```
exports.execute = function(args) {

    var vscode = require('vscode');
    var langID = vscode.window.activeTextEditor.document.languageId;
    var path = vscode.window.activeTextEditor.document.uri.fsPath.toString().replace(/\\/g, "/");

    switch (langID) {
        case 'HTML (Eex)':
            vscode.commands.executeCommand('HookyQR.beautifyFile');
            break;
        case 'vue':
            vscode.commands.executeCommand('wpyBeautify.format');
            break;
        case 'elixir':
            vimFormatter(path);
            break;
        case 'erb':
            vimFormatter(path);
            break;
        case 'ruby':
            vimFormatter(path);
            break;
        case 'NGINX':
            vimFormatter(path);
            break;
        default:
            vscode.commands.executeCommand('editor.action.formatDocument');
    }

    vscode.window.setStatusBarMessage(langID + "\t^_^", 3000);

    return 666;

    function vimFormatter(path) {
        vscode.commands.executeCommand('workbench.action.files.save');
        require('child_process').exec(`cmdow.exe /run /hid C:/opt/neovim/bin/nvim.exe -i NONE -n -E +"norm gg=G" -cwq ` + path, function puts(error, stdout, stderr) {});
    }
}
```

**Comment by mkloubert** [comment]
Does it say that my_command was not found?

How do you try to execute the command?

**Comment by mkloubert** [comment]
Oh sorry ... you defined a shortcut ...

You have to replace my_command with acro.jameel

**Comment by acrolink** [comment]
Yes, of course I define a short-cut to run `acro.jameel` but says: `command acro.jameel was not found`. It was working fine, but for unknown reason stopped.

**Comment by acrolink** [comment]
When I run `Script commands: Execute Command` get:
`[vs-script-commands] No command found that can be executed!`

**Comment by mkloubert** [comment]
I got the error that the command can be executed but the file cannot be accessed.

Try to move the script to a folder that has the same permissions as VSCode and reopen the workspace.

Do you use multi workspaces btw.?

**Comment by acrolink** [comment]
If I close VS Code and double click the `jameel.js` file in File Explorer or any file inside `my_scripts` folder, the `acrp.jameel` is working.

**Comment by mkloubert** [comment]
Does Windows ask for Admin permission? If not: Does run VSCode with Admin permission?

**Comment by acrolink** [comment]
I have tried to run in Admin mode and normal mode, same issue. No, Windows does not ask for Admin permission.

**Comment by mkloubert** [comment]
Strange ... it is working for me.

I think you use the latest release, dont you?

Does the debug output tell something?

**Comment by acrolink** [comment]
Where can I see the debug output?

**Comment by mkloubert** [comment]
`CTRL + SHIFT + Y`

If this does not help: Try to reinstall the extension:

* uninstall extension and close any VSCode instances
* keep sure to delete all `mkloubert.vs-script-commands*` folders in `C:\Users\<USERNAME>\.vscode\extensions` directory, before you ...
* open VSCode and install it again

**Comment by acrolink** [comment]
I tried, no success. I tried to move the script to another folder, no success. But if I close all VS Code instances and edit the script file (opening VS Code to edit it) itself or any file under the root script folder, it works.

**Important update**

If I want the script to get loaded and available I need to create a `.vscode` folder inside the directory where the to-be-edited file is located.

The contents of that `.vscode` folder (which I can see where I placed my custom vs code script file, and which I cannot recall how it got there in the first place) are:

`extensions.json`
`launch.json`
`settings.js`

Once I copy that `.vscode` folder to the folder where the to-be-edited file is, the script works.

**Comment by kjswanson** [comment]
The exact same issue happened to me. Working last night, and now this issue. 

I think what happened for me, is that I got it working, used it for a day or two with out closing VSC. Last night I close out of VSC. Wake up, load VSC, script commands unable to find my command.

@acrolink I wonder, do you remember if this issue happened as soon as you closed VSC for the first time after configuring vs-script-commands?
